isis/src/base/apps/framestitch/framestitch.cpp

```diff
@@ -144,7 +139,24 @@ namespace Isis {
 
       frameHeight = evenFrameHeight;
     }
+    int numFrames = evenCube->lineCount() / frameHeight;
+
+    int numLinesOverlap = 0; // The default value
+    if (ui.WasEntered("NUM_LINES_OVERLAP"))
+      numLinesOverlap = ui.GetInteger("NUM_LINES_OVERLAP");
```

Comment:
If you set an actual default of 0 for NUM_LINES_OVERLAP you don't need to worry about the default here or checking if it was entered.

Reply:
Same as below, having extra redundant checks likely can't hurt.

Reply:
I made the default to be 0 and tested that the tool can handle being run with this value not explicitly set when it then uses the default.